GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.cs

```diff
@@ -153,6 +172,7 @@ private void SetComboWidth()
         private void sortPinnedRepos_CheckedChanged(object sender, EventArgs e)
         {
             RefreshRepos();
+            SetComboWidth();
```

Comment:
❓ Rather add the call of `SetComboWidth()` to `RefreshRepos()`?
In `comboMinWidthEdit_ValueChanged`, the sequence of calls is interchanged!